12_MINI/error-codes.yaml

```diff
@@ -362,3 +362,68 @@ Errors:
   text: ""
   id: "MMU_LOAD_UNLOAD_ERROR"
   type: "CONNECT"
+- code: "12815"
+  title: ""
+  text: ""
+  id: "PRINT_FAN_ERROR"
+  type: "CONNECT"
+- code: "12816"
+  title: ""
+  text: ""
+  id: "HEATERS_TIMEOUT"
+  type: "CONNECT"
+- code: "12817"
+  title: ""
+  text: ""
+  id: "HOTEND_TEMP_DISCREPANCY"
+  type: "CONNECT"
+- code: "12818"
+  title: ""
+  text: ""
+  id: "NOZZLE_TIMEOUT"
+  type: "CONNECT"
+- code: "12819"
+  title: ""
+  text: ""
+  id: "STEPPERS_TIMEOUT"
+  type: "CONNECT"
+- code: "12820"
+  title: ""
+  text: ""
+  id: "USB_FLASH_DISK_ERROR"
+  type: "CONNECT"
+- code: "12821"
+  title: ""
+  text: ""
+  id: "HEATBREAK_THERMISTOR_FAIL"
+  type: "CONNECT"
+- code: "12822"
+  title: ""
+  text: ""
+  id: "NOZZLE_DOES_NOT_HAVE_ROUND_SECTION"
+  type: "CONNECT"
+- code: "12823"
+  title: ""
+  text: ""
+  id: "NOT_DOWNLOADED"
+  type: "CONNECT"
+- code: "12824"
+  title: ""
+  text: ""
+  id: "BUDDY_MCU_MAX_TEMP"
+  type: "CONNECT"
+- code: "12825"
+  title: ""
+  text: ""
+  id: "DWARF_MCU_MAX_TEMP"
```

Comment:
Up until now error-codes of specific printers only contained errors that are relevant on that printer. MINI does not have DWARF or MOD_BED so it should be impossible to call this warning -> Its a useless code here.

If you don't care about this in FW and need all CONNECT types in all error yamls, I get it, but be aware that it is breaking the established convention. We should definitely consider refactoring this.